### DISCUSSION TITLE
How to Access Devices Mounted on LBB as Shared Folders

### DISCUSSION THREAD

**Discussion by macmcb** [outdoorbits/little-backup-box#282] [Q&A] [answered]
I am trying to access USB storage connected to Little Backup Box as a shared folder from a Windows PC (Win10) and also via SSH. Pi 4B running Bookworm, connected to my home network by Ethernet (and wifi). I have a USB HDD plugged in and mounted as local_target via the LBB web UI. (I'm having same issue when the external drive is formatted exfat or NTFS.) On the LBB > Files page, I can see my files on the HDD under [home]/local_target. 

I am new to the Pi and Linux, so I'm not sure if my issue is related to Samba, LBB, or using Linux in general. I'm seeing two unexpected things:
 
1. On Windows Explorer, I can see folders shared by the Pi/LBB, but when I open \\raspberrypi\local_target, it is empty. It seems that this is not the same "local_target" I see when browsing Files in LBB web UI. In contrast, the "internal" folder seems to work as expected: I can create a file on LBB File Manager or on Windows and see the file appear on the other system.
2. When I log in to the Pi via SSH (Windows PowerShell) I can't see any of the mounted folders from LBB.  `lsblk` shows the connected USB device as sda1, but there is no mountpoint. In LBB, the external USB drive appears as "local_target", and a partition on the MicroSD is mounted as "internal". I have tried logging in as user "lbb" and user "pi". 

As I said, I am very new to Linux. I'm not sure if things 1 and 2 are related, but this is where my efforts to use LBB and understand Raspberry Pi have stalled out. I hope someone can point me in the right direction.

**Comment by outdoorbits** [comment, marked as answer]
You are completely right. There is a bug. Mounts are not usable any more if initiated by the web UI. Fixing this might take a few days, thank you for telling me about.

**Reply by outdoorbits** [reply]
Good news! It's working now. Please update and test it. Does it work for you?

**Reply by macmcb** [reply]
It works. Thanks for the fast response. I truly appreciate your work on this project.

**Reply by macmcb** [reply]
Update: I see the "internal" folder in Windows, but not as a mounted device with $ lsblk on the command line. I see my mounted external USB device as local_target in Windows and command line. Should I see a mountpoint for the internal folder?

Edit: I see (command line) that my files are present in /media/internal. I'm still wondering why that doesn't appear as a mountpoint, but maybe that's me not understanding Linux?

**Comment by outdoorbits** [comment]
Of course, "internal" is just a subfolder of the root partition. It's always there.
